tests/integration/redis-cli.tcl

```diff
@@ -663,6 +666,12 @@ if {!$::tls} { ;# fake_redis_node doesn't support TLS
         catch {run_cli {*}$args} output
         assert_match {*Transfer finished with success*} $output
 
+        # If sever enabled diskless sync, it should transfer the RDB by RDB channel
+        # since redis-cli sent `replconf rdb-channel 1` to ask for RDB channel.
```

Comment:
last part of the comment needs to be updated

Reply:
thank you, updated